other-docs/guides/upgrading/v14.md

```diff
@@ -59,21 +62,21 @@ In your top level project folder, add the `humanmade/workflows` framework
 package
 
 ```sh
-## Add Frameworks package
+# Add Workflows package
```

Comment:
Wanted to add a quick note this won't be a complete like-for-like swap - this module in particular has a few integrations for other modules like analytics.

Reply:
Should we add a note that there are particular workflow that requires additional modules?

Reply:
@roborourke Ping ☝🏽

Reply:
Sorry @mikelittle - I think if the workflows module is gone, then yeah perhaps updating the analytics module to check for the existence of the Workflows plugin instead is the way to go.

Reply:
Actually, I suppose we want to be removing any traces of dependency or support for things that are no longer bundled? Maybe the approach is to move the workflows code out of the analytics module to a gist and reference that saying, "If you are using Workflows and Altis Analytics you can find the legacy integration code for analytics notifications here".

What do you think?

Reply:
Yep. I can do that.

By the way, is Analytics staying in Altis cloud now? Or is this just until Accelerate replaces it?

Reply:
I’m pretty it’s staying but deprecated, until removal in v15. I’d check with @veselala

Reply:
as per my last conversation with @rmccue we will bee keeping Accelerate functionalities in Altis till v15, as we need to communicate and migrate our current customers to Accelerate first.

Reply:
I have removed the workflow code from Altis/Analytics PR (https://github.com/humanmade/altis-analytics/pull/376)